src/pages/tBTC/Bridge/TbtcBalanceCard.tsx

```diff
@@ -1,25 +1,18 @@
 import { FC, ComponentProps } from "react"
 import TokenBalanceCard from "../../../components/TokenBalanceCard"
 import { Token } from "../../../enums"
-import { Card, Box } from "@threshold-network/components"
+import { Card } from "@threshold-network/components"
 
 export const TbtcBalanceCard: FC<ComponentProps<typeof Card>> = ({
   ...restProps
 }) => {
   return (
     <TokenBalanceCard
       token={Token.TBTC}
-      title={
-        <>
-          <Box as="span" textTransform="lowercase">
-            t
-          </Box>
-          btc balance
-        </>
-      }
+      title="tBTC Balance"
       tokenSymbol={"tBTC"}
+      higherPrecision={2}
```

Comment:
I've made a mistake and we actually should use 6 here based on this:

https://github.com/threshold-network/bitcoin-on-base/blob/70e7582866a3aff92066b85eec4f2872811eaedb/src/components/TokenBalance.tsx#L40

```suggestion
      higherPrecision={6}
```

Reply:
Ref commit: ee97ceac